benchmarks/dynamo/common.py

```diff
@@ -3001,7 +3001,8 @@ def main(runner, original_dir=None):
                 f"--diff-branch: current branch is same as {args.diff_branch} branch, what are you diffing?"
             )
 
-    if args.multiprocess:
```

Comment:
another thing i'm changing regarding parent processes

currently, when --multiprocess is set, we spawn too many processes:
```
parent -> mp.spawn child_i -> subprocess spawn child_j for model (--only) ->  mp.spawn -> child_k -> init dist rank_l -> ERROR: multiple dist init
```

once we change to only mp.spawn in child processes:
```
parent -> subprocess spawn child_i for model (--only) ->  mp.spawn child_j -> init dist rank_k -> calls model
```

but to do this we need to change CUDA_VISIBLE_DEVICES overriding logic